### DISCUSSION TITLE
Clarity on Error definitions

### DISCUSSION THREAD

**Discussion by globalflea** [serverlessworkflow/specification#497] [Q&A] [answered]
In the latest (0.7.x) the error definition object (at the workflow) is given as a sample json with the following fields:
- name, code and description

however, there it is not clear (at least to me) if this is the complete list or just a sample.  With the other definitions, a table with "Parameter", "Description", "Type", "Required" is provided. Will this also be done for Error Definitions?

was defined in v0.6 with the following fields:
- error, code, retry_ref, transition_or_end

but i couldn't find this in the recent specs

**Comment by tsurdilo** [comment, marked as answer]
Good find, it's missing. Will add to https://github.com/serverlessworkflow/specification/blob/main/specification.md#workflow-error-handling

**Comment by tsurdilo** [comment]
@globalflea note that if a certain definition is missing in the spec doc (which needs to be added then, so keep finding those issues :) ), the "source of truth" for the DSL is the workflow JSON Schemas https://github.com/serverlessworkflow/specification/blob/main/schema/

specifically for error def, you can reference https://github.com/serverlessworkflow/specification/blob/main/schema/errors.json#L29

**Comment by tsurdilo** [comment]
https://github.com/serverlessworkflow/specification/issues/498
